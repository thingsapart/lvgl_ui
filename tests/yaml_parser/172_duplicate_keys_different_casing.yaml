# Keys are case sensitive, so these are not duplicates
Key: 1
key: 2
