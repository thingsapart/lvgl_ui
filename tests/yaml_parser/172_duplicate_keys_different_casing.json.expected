[{
		"Key":	1,
		"key":	2
	}]
